bundles/org.openhab.binding.sncf/README.md

```diff
@@ -0,0 +1,70 @@
+# SNCF Binding
+
+The SNCF binding provides real-time data for each train, bus, tramway... station in France.
+This is based on live API provided by DIGITALSNCF.
+
+Get you API key at https://www.digital.sncf.com/startup/api/token-developpeur
```

Comment:
Plase add a link, for example 
`Get you API key on [DIGITALSNCF web site](https://www.digital.sncf.com/startup/api/token-developpeur)`